src/assets/env.js

```diff
@@ -0,0 +1,11 @@
+window.appSettings = {};
+
+window.appSettings.environmentDisplayName = 'Staging site';
+window.appSettings.apiUrl = 'https://dsgocdnapi.azureedge.net/';
+window.appSettings.adminUrl = 'https://dsgocdnapi.azureedge.net/';
+window.appSettings.fdrsAuth = '';
+window.appSettings.mbtoken = 'pk.eyJ1IjoiZ28taWZyYyIsImEiOiJjams3b2ZhZWswMGFvM3hxeHp2ZHFhOTRrIn0._pqO9OQ2iNeDGrpopJNjpg';
+window.appSettings.showEnvBanner = 'false';
```

Comment:
Is it required to make it string? We can simply use `false` (boolean value) instead of `'false'` (string)